packages/usa-input-mask/src/styles/_usa-input-mask.scss

```diff
@@ -0,0 +1,26 @@
+@use "uswds-core" as *;
+
+.usa-input-mask {
+  position: relative;
+
+  &--content {
+    border: 1px solid transparent;
+    color: color("gray-50");
+    font-size: font-size($theme-form-font-family, $theme-body-font-size);
+    left: 2px;
+    pointer-events: none;
+    position: absolute;
+    top: 0;
+
+    i {
+      visibility: hidden;
+    }
+  }
+}
+
+.usa-masked,
+.usa-input-mask--content {
+  @extend %block-input-general;
```

Comment:
More on alignment: I am wondering if we should switch to the mono font for this component. It would be a visual deviation from our other text input but would ensure that the characters of the input content lines up exactly with the mask content.

Reply:
I've done a lot of back-and-forth on this, and at the end of the day I don't think we're gaining as much from the font change as I'd hoped. In fact, the monospaced `space` character can make it look as though you have an extra space added in your entry. So I think the best option is to stick with the conventional font here.